### PR TITLE
created lose method an verify if there is enough space to put the shape

### PR DESCRIPTION
**Pull request by jpgonzales27**

### CODE REVIEW

File: app/src/main/java/src/shapes/Shape.java [hunk 1]
```diff
@@ -41,31 +41,43 @@ public void moveRight() {
             xPosition += 1;
         }
     }
-
+    /**
+     * @return if container is at left limit.
```
Comment (JessickaMoya) [inline app/src/main/java/src/shapes/Shape.java]: Your javadoc is  not according to the method

File: app/src/main/java/src/shapes/Shape.java [hunk 2]
```diff
@@ -41,31 +41,43 @@ public void moveRight() {
...
+     */
     public int getDownRows() {
         return downRows;
     }
-
+    /**
+     * @return if container is at left limit.
```
Comment (JessickaMoya) [inline app/src/main/java/src/shapes/Shape.java]: Your javadoc is  not according to the method

File: app/src/main/java/src/shapes/Shape.java [hunk 3]
```diff
@@ -41,31 +41,43 @@ public void moveRight() {
...
+     */
     public int getLeftColumns() {
         return leftColumns;
     }
-
+    /**
+     * @return if container is at left limit.
```
Comment (JessickaMoya) [inline app/src/main/java/src/shapes/Shape.java]: Your javadoc is  not according to the method

File: app/src/main/java/src/shapes/Shape.java [hunk 4]
```diff
@@ -41,31 +41,43 @@ public void moveRight() {
...
+     */
     public boolean checkDownLimit() {
         return yPosition == verticalLimit - container.length + downRows;
     }
-
+    /**
+     * @return if container is at left limit.
```
Comment (JessickaMoya) [inline app/src/main/java/src/shapes/Shape.java]: Your javadoc is  not according to the method

File: app/src/main/java/src/shapes/Shape.java [hunk 5]
```diff
@@ -41,31 +41,43 @@ public void moveRight() {
...
+     */
     public int getRightColumns() {
         return rightColumns;
     }
-
+    /**
+     * @return if container is at left limit.
```
Comment (JessickaMoya) [inline app/src/main/java/src/shapes/Shape.java]: Your javadoc is  not according to the method

File: app/src/main/java/src/shapes/Shape.java (hunk 1 above)
Comment (jpgonzales27) [reply app/src/main/java/src/shapes/Shape.java]: you're right jessi, i'll change it

**Review by JRojas999 — APPROVED** [review]

File: app/src/main/java/src/shapes/Shape.java (hunk 1 above)
Comment (jpgonzales27) [reply app/src/main/java/src/shapes/Shape.java]: I already changed it Jessi, could you check it please

**Review by JessickaMoya — APPROVED** [review]

**Review by Raymundogs — APPROVED** [review]

**Pull request merged**